two-data-centers-in-one-city-deployment.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Two Data Centers in One City Deployment
+summary: Learn the deployment solution to two data centers in one city.
+---
+
+# Two Data Centers in One City Deployment with DR Auto-Sync Mode
+
+This document introduces the deployment mode, architecture, and configuration of the two data centers (DC) in one city, as well as how to enable this delpoyment mode and to use the replicas in this mode.
+
+In an on-premises environment, TiDB usually adopts the multi-data-center deployment mode to ensure high availability and disaster recovery capability. The multi-data-center deployment mode includes a variety of deployment modes, such as three data centers in two cities and three data centers in one city. This document introduces the deployment mode of two data centers in one city. With lower cost, TiDB can also meet the requirements of high availability and disaster recovery. The deployment mode adopts Data Replication Auto Synchronous mode, or DR Auto-Sync mode.
+
+Under the mode of two data centers in one city, the distance between the two data centers is within 50 kilometers. They are usually located in the same city or two adjacent cities (such as Beijing and Langfang in Hebei province). The network latency between the data centers is lower than 1.5 milliseconds and the bandwidth is higher than 10 Gbps.
+
+## Architecture
+
+This section takes the example of a city where two data centers IDC1 and IDC2 are located separately in the east and west.
+
+The architecture of the cluster deployment is as follows:
+
+- The TiDB cluster is deployed to two DCs in one city: the primary IDC1 in the east, and the DR IDC2 in the west.
+- The cluster has 4 replicas, 2 Voter replicas in IDC1, 1 Voter replicas and 1 Learner replicas in IDC2. For the TiKV component, each rack has a proper label.
+- The Raft protocol is adopted to ensure consistency and high availability of data, which is transparent to users.
+
+![2-DC-in-1-city architecture](/media/two-dc-replication-1.png)
```

Comment:
the picture is missing.

Reply:
fixed in https://github.com/pingcap/docs/pull/6182/commits/2552ec204a9da9faf59caa8ce834e9210a7fae54.